Sevenpros/Teamwork
# #168373597 Build out new article area

#### What does this PR do?
Build out new article area
#### Description of Task to be completed?
create user.html
add navigation bar with team work logo
split the main tab into 3 tabs
display user profile info
add text area for new article
add icons for uploading images and video
add post button
#### How should this be manually tested?
when a employee successfully logged in the app he/she should be able to see a text area 
to write in new article post
the post button should be clear
comment button are also available to upload images and videos

#### What are the relevant pivotal tracker stories?
#168373597
